doc/src/modules/physics/continuum_mechanics/beam_problems.rst

```diff
@@ -336,41 +336,38 @@ applied from the mid till the end of the beam.
 >>> b.apply_load(R1, 0, -1)
 >>> b.apply_load(1.5, 3, -2)
 >>> b.apply_load(-3, 3, 0)
->>> b.apply_load(-1, 3, 1)
+>>> b.apply_load(-1/3, 3, 1)
 >>> b.apply_load(R2, 6, -1)
 >>> b.bc_deflection.append((0, 0))
 >>> b.bc_deflection.append((6, 0))
 >>> b.solve_for_reaction_loads(R1, R2)
 >>> b.reaction_loads
-    {R₁: 2.75, R₂: 10.75}
+    {R₁: 2.25, R₂: 8.25}
+   
 >>> b.load
-            -1              -2            0          1                -1
-    2.75⋅<x>   + 1.5⋅<x - 3>   - 3⋅<x - 3>  - <x - 3>  + 10.75⋅<x - 6>
+            -1              -2            0                            1               -1
+    2.25⋅<x>   + 1.5⋅<x - 3>   - 3⋅<x - 3>  - 0.333333333333333⋅<x - 3>  + 8.25⋅<x - 6>  
```

Comment:
This looks terrible...why are we not using rationals? or round?

Reply:
I agree it is preferable to avoid floats.

Reply:
> This looks terrible...why are we not using rationals? or round?

This might be the most opiniated statement I've seen from you in 10 years! :)

Reply:
LOL. I apologize if it seemed too strident. I would have said the same thing if I had done it myself. But lacking ability to convey tone of voice I should have simply rephrased this as "Would it be possible to use `round` or `Rationals` to make this look tidier?" (I admit that "blah blah clode is messy" always stings a bit, too. Obviously, we all grow with time and anyone who makes such statements will likely be guilty of doing the same to some degree...in someone else's eyes. So best to leave the stones in the pile and get out the Windex to help with your neighbor's glass walls.)

Thanks for saying something, @moorepants .

So...back to this example. Is this just a custom printer to order terms of a polynomial? And did the floats bleed through because "floats in floats out"?

Reply:
I didn't take it in a bad way and yes I likely imagined more tone than you probably did when righting. I just thought it was funny, to be honest :)

It isn't a custom printer. These are so call singularity functions, which are polynomials that can be "turned on" at specific values. And yes, floats in floats out.

Reply:
Ah. Then pox on the caller! :-) I don't know much about these SymPy objects. 

...looking...

OK, so what we are seeing here is just the `pprint` form of the expression. And the user has the normal options of evaluating to a given number of sig figs or (with a little more effort) of a given number of digits:
```
>>> eq= SingularityFunction(x, 4, 4)/3
10*SingularityFunction(x, 4, 4)/3
>>> eq.replace(
... lambda x:x.is_Mul and x.args[0].is_Number,
... lambda x:x.args[0].round(2)*Mul(*x.args[1:]))
3.33*SingularityFunction(x, 4, 4)
>>> eq.n(2)
3.3*SingularityFunction(x, 4, 4)
>>> pprint(_)
           4
3.3⋅<x - 4> 
```